app/src/main/java/org/simple/clinic/patient/filter/WeightedLevenshteinSearch.kt

```diff
@@ -3,7 +3,6 @@ package org.simple.clinic.patient.filter
 import info.debatty.java.stringsimilarity.CharacterInsDelInterface
```

Comment:
I had no idea, this class was a thing 😮 . Is this ranking for results?

Reply:
Yup